site/styles/_variables.scss

```diff
@@ -248,3 +248,10 @@ $font-family-base: 'Poppins', system-ui, -apple-system, 'Segoe UI', Roboto, 'Hel
 $font-weight-light: 300 !default;
 $font-weight-normal: 400 !default;
 $font-weight-bold: 500 !default;
+
+// Tooltip
+
+$tooltip-color: var(--body-bg, #000) !default;
```

Comment:
Can you swap these for CSS variables please.  All new variables should be ideally be CSS variables instead of Sass variables and definitely not Bootstrap variables.

Reply:
- changed to css vars